src/app/application.cpp

```diff
@@ -660,4 +661,7 @@ void Application::validateCommandLineParameters()
 {
     if (m_commandLineArgs.portableMode && !m_commandLineArgs.profileDir.isEmpty())
         throw CommandLineParameterError(tr("Portable mode and explicit profile directory options are mutually exclusive"));
+
+    if (m_commandLineArgs.portableMode && m_commandLineArgs.relativeFastresumePaths)
+        throw CommandLineParameterError(tr("Portable mode and explicit relative fastresume mode options are mutually exclusive"));
```

Comment:
Well, if they read the documenation they should have that impression. But I am ok with a warning.